src/main/scala/zio/prelude/Invariant.scala

```diff
@@ -49,6 +52,15 @@ object Invariant {
         Equivalence(_.map(f.to), _.map(f.from))
     }
 
+  implicit val FutureInvariant: Invariant[Future] =
+    new Invariant[Future] {
+      def invmap[A, B](f: A <=> B): Future[A] <=> Future[B] =
+        Equivalence(
+          fa => zio.Runtime.default.unsafeRunToFuture(ZIO.fromFuture(_ => fa).map(f.to)),
```

Comment:
Can't we just `map` here instead of unsafely running? If we need to we can make it an `implicit def` and require an `ExecutionContext` to be in scope.

Reply:
Yeah, that was my initial approach, but wasn't sure about it